sdk/storage/storage-blob/package.json

```diff
@@ -21,7 +21,7 @@
   },
   "scripts": {
     "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
-    "build:autorest": "autorest ./swagger/README.md --typescript --use=@microsoft.azure/autorest.typescript@5.0.1",
+    "build:autorest": "autorest ./swagger/README.md --typescript --package-version=12.0.1 --use=@microsoft.azure/autorest.typescript@5.0.1",
```

Comment:
>version [](start = 75, length = 7)

minor: About the versioning update, would you help to update the checklist(One note) we used for pre-release check? :)

Reply:
Will do